src/librustc_arena/lib.rs

```diff
@@ -602,62 +602,73 @@ macro_rules! which_arena_for_type {
 
 #[macro_export]
 macro_rules! declare_arena {
-    ([], [$($a:tt $name:ident: $ty:ty, $gen_ty:ty;)*], $tcx:lifetime) => {
+    ([], [$($a:tt $name:ident: $ty:ty, $_gen_ty:ty;)*], $tcx:lifetime) => {
```

Comment:
why didn't you remove this unused parameter? Is this macro used as input to another macro? If so, please leave a comment